doc/examples/qti_simulation_tutorial.py

```diff
@@ -0,0 +1,137 @@
+"""
+============================
+Q-SpaceTrajectory Simulation
+============================
+
+The most basic assumption of qti states that tissues can be denoted by multiple gaussian component.
+QTI can only handle a single gtab at a time, so in this tutorial we'll start by taking 2 gtabs then combine them into a single btensor. 
+"""
+
+import matplotlib.pyplot as plt
+import numpy as np
+import math
+
+from dipy.reconst.tests.test_qti import  _anisotropic_DTD, _isotropic_DTD 
+from dipy.core.gradients import gradient_table 
+from dipy.core.sphere import disperse_charges, Sphere, HemiSphere
+from dipy.sims.voxel import multi_tensor
+
+"""
+We'll take the case of 2 gtabs and then combine them into a single btensor as for qti model, we need a single gtab.
+"""
+
+""" 
+Creating the first gtab 
+""" 
+
+n_pts = 20
+theta = np.pi * np.random.rand(n_pts) 
+phi = 2 * np.pi * np.random.rand(n_pts)
+hsph_initial = HemiSphere(theta=theta, phi=phi)
+hsph_updated, potential = disperse_charges(hsph_initial, 5000) 
+bvecs1 = np.concatenate([hsph_updated.vertices] * 4)
```

Comment:
Name this variable as bvecs and not bvecs1 and say that this will correspond to the main directions of the btensors.